index.rst

```diff
@@ -29,67 +27,260 @@ Guidelines for User Tutorials
+**Referenced.**
+Provide the user with links to external resources (e.g., code package documentation, papers).
+Mention any precursor (or advanced) tutorials that users can consider as prerequisite (or as follow-up).
 
-**Properly-credited:**
-Appropriate acknowledgments should be provided to credit individuals whose work was used.
-This sets a precedent of prioritizing credits in an openly collaborative environment.
-Cite other scientists or papers within the text of the tutorial where appropriate.
+**Credited.**
+Individuals who contributed to the tutorial, or whose work was used as a basis or inspiration for the tutorial, must be acknowledged by name.
+Include paper citations where possible.
 
-**Clearly-written:**
-Follow the best practices described under :ref:`Narrative text<narrative-text>`.
-Write short, clear, instructional statements in the `imperative mood <https://en.wikipedia.org/wiki/Imperative_mood>`_.
 
+.. _tutorial-types:
 
-.. _how-to-contribute:
+Tutorial types
+==============
 
-How to contribute
-=================
+Every tutorial, whether a Jupyter Notebook or a documentation-based tutorial,
+must choose a single type from the three options below,
+be designed to fit into the tutorials' :ref:`organizational hierarchy <tutorial-types-org>`,
+and follow the :ref:`naming convention <tutorial-types-names>`.
+
+
+.. _tutorial-types-howto:
+
+How-to: RSP functionality
+-------------------------
+
+Short tutorials with a learning objective of how to use a single RSP function or tool.
+
+These tutorials are typically written without scientific context or motivation - just simple examples for quick reference.
+
+Each "How-to" tutorial should take about 10 minutes to work through.
+
+
+.. _tutorial-types-prod:
+
+Data product definitions
+------------------------
+
+Tutorials with a learning objective of understanding one of the data products for a given release.
+
+These tutorials each focus on one data product, e.g., one image type, one map type, or one catalog table.
+They provide a description of the data and metadata, and demonstrate how to access the data product -- via multiple methods (e.g., TAP and butler) where applicable.
+
+Descriptions of the data and metadata can be mainly qualitative (i.e., primary columns, recommended flags) but should include key quantities such as counts, sizes, volumes, coverage, etc.
+These tutorials should link to the appropriate section of the data release documentation.
+
+
+.. _tutorial-types-sci:
+
+Science demonstrations
+----------------------
+
+Longer tutorials with a learning objective of executing an end-to-end scientific workflow.
+
+These tutorials demonstrate data visualization techniques and scientific analyses that use multiple RSP tools and data products in sequence.
+They should describe key scientific concepts and include scientific context and motivation for why the data and tools are used.
+
+
+.. _tutorial-types-org:
+
+Organizational hierarchy
```

Comment:
Would it make sense for this section to be a level higher in the TOC tree? (i.e., the same level as "Tutorial types"?)

Reply:
I'll just leave it there. It's the organizational hierarcy *for* the tutorial types.